opencti-platform/opencti-graphql/src/http/httpChatbotProxy.ts

```
@@ -0,0 +1,88 @@
+import nconf from 'nconf';
+import axios from 'axios';
+import type Express from 'express';
+import { createAuthenticatedContext } from './httpAuthenticatedContext';
+import { getEntityFromCache } from '../database/cache';
+import { ENTITY_TYPE_SETTINGS } from '../schema/internalObject';
+import { getEnterpriseEditionInfo } from '../modules/settings/licensing';
+import { logApp } from '../config/conf';
+import type { BasicStoreSettings } from '../types/settings';
+import { setCookieError } from './httpUtils';
+
+export const getChatbotProxy = async (req: Express.Request, res: Express.Response) => {
+  try {
+    const context = await createAuthenticatedContext(req, res, 'chatbot');
+    if (!context.user) {
+      res.sendStatus(403);
+      return;
+    }
+
+    const chatbotUrl = nconf.get('ai:chatbot:url');
+    const isChatbotEnabled = nconf.get('ai:chatbot:enabled');
+    const settings = await getEntityFromCache<BasicStoreSettings>(context, context.user, ENTITY_TYPE_SETTINGS);
+    const { license_raw_pem } = getEnterpriseEditionInfo(settings);
+    if (!isChatbotEnabled || !license_raw_pem) {
+      return;
```

We'll probably want to return an error in this case, but I'm not sure which one yet. We can already send a 400 error even if we'll need to discuss further the error message